build-system/global-configs/canary-config.json

```diff
@@ -6,6 +6,7 @@
     "inabox-css-cleanup"
   ],
   "allow-url-opt-in": [
+    "ios-scrollable-iframe",
```

Comment:
I think we're disallowing new URL opt-ins? @mrjoro

Reply:
We've deprecated support for document-level experiments in favor of origin trials per #15827.  See that issue for more of the reasoning, but the basic idea is that we want to have more knowledge of who is opting into them so that we can let them know if we are disabling/removing the experimental feature.

I'm not familiar with the URL opt in.  Do we have documentation for it?

Reply:
URL opt-in was mostly used for viewer/document experiment propagation where both viewer and document should be in sync. Rare, but sometimes necessary.

Reply:
I spoke to @dvoytenko offline, and it sounds like these are used to enable experiment from the viewer, and publishers wouldn't be able to actually enable it themselves.  In that case, I think these are okay.